features/include/pcl/features/impl/ppf.hpp

```diff
@@ -94,10 +94,8 @@ pcl::PPFEstimation<PointInT, PointNT, PointOutT>::computeFeature (PointCloudOut
           Eigen::Affine3f transform_mg (Eigen::Translation3f ( rotation_mg * ((-1) * model_reference_point)) * rotation_mg);
 
           Eigen::Vector3f model_point_transformed = transform_mg * model_point;
-          float angle = std::atan2 ( -model_point_transformed(2), model_point_transformed(1));
-          if (std::sin (angle) * model_point_transformed(2) < 0.0f)
-            angle *= (-1);
-          p.alpha_m = -angle;
+          float angle = std::atan2 (model_point_transformed(2), model_point_transformed(1));
+          p.alpha_m = angle;
```

Comment:
Did you forget a minus sign here?

And I am still not sure about +pi becoming -pi. Even if they represent the same angle, couldn't this cause a problem in the discretization in PPFRegistration?